src/shogun/classifier/mkl/MKL.cpp

```diff
@@ -278,7 +278,11 @@ void CMKL::register_params()
 	SG_ADD(&mkl_block_norm, "mkl_block_norm", "mkl sparse trade-off parameter", MS_NOT_AVAILABLE);
 
 	m_parameters->add_vector(&beta_local, &beta_local_size, "beta_local", "subkernel weights on L1 term of elastic net mkl");
-	watch_param("beta_local", &beta_local, &beta_local_size);
+	watch_param(
+	    "beta_local", &beta_local, &beta_local_size,
+	    AnyParameterProperties(
+	        "subkernel weights on L1 term of elastic net mkl", MS_NOT_AVAILABLE,
```

Comment:
same here, pls let's not have this `MS_NOT_AVAILABLE, GRADIENT_NOT_AVAILABLE` everytime, but instead hide it, and only write it out if things are available